src/v/crypto/tests/BUILD

```diff
@@ -147,3 +147,59 @@ redpanda_cc_bench(
         "@seastar//:benchmark",
     ],
 )
+
+redpanda_test_cc_library(
+    name = "ossl_context_service_test_base",
+    hdrs = [
+        "ossl_context_service_test_base.h",
+    ],
+    visibility = ["//visibility:private"],
+    deps = [
+        "//src/v/ssx:thread_worker",
+        "//src/v/test_utils:gtest",
+        "@googletest//:gtest",
+    ],
+)
+
+redpanda_cc_gtest(
+    name = "ossl_context_service_test",
+    timeout = "short",
+    srcs = [
+        "ossl_context_service_test.cc",
+    ],
+    cpu = 1,
```

Comment:
chore: This ran with 2 CPUs in CMake - that was to ensure there wasn't any cross thread poisioning with the library contexts

Reply:
oh, great catch. thanks.

Reply:
done